apps/app-loader/spawn/src/main.rs

```diff
@@ -0,0 +1,161 @@
+#![no_std]
+#![no_main]
+
+enum StartupCommand {
+    Unhandled = 0,
+    LoadElf = 1,
+    PingResponse = 2,
+}
+
+impl From<xous::MessageId> for StartupCommand {
+    fn from(src: xous::MessageId) -> StartupCommand {
+        match src {
+            1 => StartupCommand::LoadElf,
+	    2 => StartupCommand::PingResponse,
+            _ => StartupCommand::Unhandled,
+        }
+    }
+}
+
+#[panic_handler]
+fn handle_panic(arg: &core::panic::PanicInfo) -> ! {
+    log::info!("{arg}");
+    loop {}
```

Comment:
Instead of busy-looping here, the program should quit via `xous::terminate_process(1)`.